browser/css/toolbar.css

```diff
@@ -1016,7 +1016,7 @@ button.leaflet-control-search-next
 .w2ui-icon.link{ background: url('images/lc_inserthyperlink.svg') no-repeat center; }
 .w2ui-icon.insertsymbol{ background: url('images/lc_insertsymbol.svg') no-repeat center; }
 .w2ui-icon.edit{ background: url('images/lc_editdoc.svg') no-repeat center; }
-.w2ui-icon.fold{ background: url('images/fold.svg') no-repeat center center / 19px; opacity: 0.8; filter: alpha(opacity=80);}
```

Comment:
isn't this change already present in https://github.com/CollaboraOnline/online/pull/9900 ? Shouldn't we just merge that first?